src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -595,12 +595,15 @@ private async Task<AuthenticationResponse> AuthenticateWithPasswordAsync(Authent
                 if (!identifyResponse.IsLoginSuccess)
                 {
                     // Verify if password expired
-                    if (IsRemediationRequireCredentials(RemediationType.ReenrollAuthenticator, identifyResponse))
+                    if (IsRemediationRequireCredentials(RemediationType.ReenrollAuthenticator, identifyResponse) 
+                        || IsRemediationRequireCredentials(RemediationType.ReenrollAuthenticatorWarning, identifyResponse))
                     {
                         return new AuthenticationResponse
                         {
                             AuthenticationStatus = AuthenticationStatus.PasswordExpired,
                             IdxContext = idxContext,
+                            Messages = identifyResponse.IdxMessages?.Messages,
+                            CanSkip = identifyResponse.ContainsRemediationOption(RemediationType.Skip),

```

Comment:
Is there a technical reason we don't update the CreateAuthenticationResponse method to include these bits for consistency, and then use a call to CreateAuthenticationResponse instead.  I think it would be preferable to set the `Messages` property whenever we need to return a new AuthenticationResponse.

https://github.com/okta/okta-idx-dotnet/blob/fa270f61a476b2803e2a7dab0402601a9459e7de/src/Okta.Idx.Sdk/IdxClient.cs#L239

Reply:
Good point, I'll make sure to update the method to include Messages.